DOC-4384: [BRANDING-1] Enhance eXo Branding to make it more extensible (#217)
exoplatform/exo-documentation: docs/Administration.rst

Branding eXo Platform
======================

> eXo Platform allows you to easily brand **eXo Platform** by changing your company logo, the color of the top navigation bar in the portal and picking colors for your buttons, titles, etc..

Reviewer: ```suggestion
eXo Platform can be easily branded by setting your company logo, changing the color of the top navigation bar in the portal and picking colors for your buttons, titles, etc..
```

> 1. Click |image0| --> OTHER --> Branding on the side navigation bar.

Reviewer: ```suggestion
1. Click |image0| --> OTHER --> Branding on the side navigation panel.
```

The Branding page will be displayed:

|image1|

**Details**:

> -  The Company Name pane: Uploads your company name to be displayed in the topbar and will be used in the left side of the top navigation bar.

Reviewer: ```suggestion
-  The Company Name pane: Uploads your company name to be displayed in the top bar.
```

-  The Select Logo pane: Uploads your company logo that must be in the
   ``.png`` format, and with Recommended size **300x34** and a transparent background.

> -  The Primary, Secondary and Tertiary colors of the Digital Workplace branding.

Reviewer: ```suggestion
-  The Primary, Secondary and Tertiary colors of the user interface.
```

-  The Select Navigation Bar Style pane: Selects the navigation bar
   style suitable to your logo. There are two styles available on the
   portal, consisting of Light which is default, and Dark.

> .. note::  As default values, the default Company name is **COMPANY** and a default logo are predefined in the Digital Workplace.

Reviewer: ```suggestion
.. note::  As default values, the default Company name is **COMPANY** and a default logo are predefined.
```

.. note::  Even if you delete your company logo, the default logo will be displayed..

3. Click on Primary, Secondary and Tertiary color pickers so that your style selection from a color picker will also make the color 
   of buttons and backgournd.

|image364|

> You can use the color picker right to select a color and get the HTML Color Code of this pixel. Also you get the HEX color code value of the selected color or put on your HEX and it will be automatically identified, or even you can select a predefined color from the palette.

Reviewer: ```suggestion
You can use the color picker right to select a color and get the HTML color code of this pixel. Also you get the hexadecimal color code value of the selected color or directly enter the hexadecimal code and it will be automatically recognized. Finally, you can also select a predefined color from the palette.
```

> .. note::  -  Primary color Call to action color, Emphasized titles, text hover or press, hyper-links or selected links (Primary and text buttons, Notifications, Actions icon buttons, Spaces and avatars names, mention tags, etc..)

Reviewer: ```suggestion
.. note::  -  Primary color is used for main call to actions, emphasized titles, text hover or press, hyper-links or selected links (Primary and text buttons, Notifications, Actions icon buttons, Spaces and avatars names, mention tags, etc..)
```

.. note::  Even if you delete your company logo, the default logo will be displayed..

> 3. Click on Primary, Secondary and Tertiary color pickers so that your style selection from a color picker will also make the color 

Reviewer: ```suggestion
3. Click on Primary, Secondary and Tertiary color pickers to set the color of buttons and background
```

Reviewer: this should be a bit more developed. Give at least two usecases for each color in eXo default skin.

Reviewer: also you could add a screnehsot of the picker and say the value is the hexadecimal code of the color

>    of buttons and backgournd.

Reviewer: ```suggestion
  
```
